app/views/components/docs/download-link.yml

```diff
@@ -0,0 +1,16 @@
+name: Download link
+description: A link with a file download icon
+accessibility_criteria: |
+  The download link must:
+  - accept focus
+  - be focusable with a keyboard
+  - be usable with a keyboard
+  - indicate when it has focus
+  The download icon must:
+  - be presentational and ignored by screen readers
+fixtures:
+  default:
+    href: '/download-this'
+  custom_link_text:
+    href: '/download-map'
+    download_link_text: 'Download Map (PDF)'
```

Comment:
This file should have a new line at the end. This is something you can configure in your editor.